Packs/ZeroFox/ReleaseNotes/1_3_1.json

```
@@ -0,0 +1,4 @@
+{
+    "breakingChanges": true,
+    "breakingChangesNotes": "The zerofox-get-alert-attachments command was deleted to avoid confusion from customers on its usage."
```

please use the same comment in the rn, it's more clear
```suggestion
    "breakingChangesNotes": "Deprecated the ***zerofox-get-alert-attachments*** command due to unintended behavior."
```